migrations/008_add_object_filter_ratio.py

```diff
@@ -0,0 +1,38 @@
+"""Peewee migrations -- 007_add_object_filter_ratio.py.
```

Comment:
Need to update this comment with the new number and filename

Reply:
My PR which had a new migration in it just got merged so you'll need to update this file to `009` as well